RTClib.h

```diff
@@ -231,6 +234,9 @@ class TimeSpan {
 
 protected:
   int32_t _seconds; ///< Actual TimeSpan value is stored as seconds
+  static constexpr int32_t SECS_PER_MIN = 60; ///< Number of seconds in a minute
+  static constexpr int32_t MINS_PER_HOUR = 60; ///< Number of minutes in an hour
+  static constexpr int32_t HOURS_PER_DAY = 24; ///< Number of hours in a day
```

Comment:
How would you feel about making these either global or public?

Reply:
I have no strong feelings either way: they certainly look like generally useful, but they are kind of trivial, and they would risk collisions in the global namespace. Maybe public would be OK. I guess you should get the opinion of a maintainer: I am just a random contributor.

Reply:
Maybe a static (constexpr) global then? That way they won't cause global namespace collisions. This library as-is has the magic number antipattern. Fixing it is out of scope for this PR though.

Reply:
On second thought, `static constexpr`s won't work in this case because they are in the header. Statics in a header are textually `#include`d, and `static` means limited to a "translation uint", so the constants would still be exposed in the public interface.